NuGet.Config

```diff
@@ -2,6 +2,13 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 
 <configuration>
+  <packageRestore>
+    <!-- Currently, the repository's version of NuGet.exe and Visual Studio's version
+         fight over the format of project.lock.json because the one in the respository
+         is newer. To prevent that, turn off package restore. -->
```

Comment:
What bug is tracking the eventual removal of this?

@emgarten: this argues against your model of "use v2 if there's no reason not to". Is there an alternative flag we can set to force v1 in the automatic restore?